superdesk/web-publisher
# [Theme][WebRendererBundle] SWP-200 Asset function is not theme-aware

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | [SWP-200](https://dev.sourcefabric.org/browse/SWP-200)
| License       | AGPLv3

This makes the assets theme-aware. Every asset should be stored under the public directory. See [theme structure documentation](http://superdesk-web-publisher.readthedocs.io/en/latest/themes.html#structure) for more details. The default implementation of SyliusThemeBundle allows only to install assets for bundles. This PR makes it possible to install assets out of the bundle scope.

**How to place and install assets:** (an example)
1. Put the `test.css` asset file in `<theme-name>/public/css/` (create css directory if not exists)
2. Install assets by running command: `php app/console sylius:theme:assets:install`
3. Make use of the asset file in twig template:

```twig
<link rel="stylesheet" href="{{ asset('bundles/css/test.css') }}" />
```

**To see this PR in action run:**

```bash
php app/console theme:setup -f
php app/console sylius:theme:assets:install
```
